src/module/test/item-counter.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @jest-environment jsdom
+ */
+import count from '../count-items.js';
+
+document.body.innerHTML = `
+ <div class="home"><div class="meal">img</div></div>
+ `;
+
+describe('Add new meals and count it', () => {
+  test('if jest work correctly', () => {
+    expect(true).toBe(true);
+  });
+
+  test('Add meal-1', () => {
+    const parent = document.querySelector('.home');
+    const row = document.createElement('div');
+    row.innerHTML = 'mealsName';
+    parent.appendChild(row);
+
+    expect(count(parent)).toBe(2);
+  });
+
+  test('Add meal-2', () => {
+    const parent = document.querySelector('.home');
+    const row = document.createElement('div');
+    row.innerHTML = 'mealsName';
+    parent.appendChild(row);
+
+    expect(count(parent)).toBe(3);
+  });
+});
```

Comment:
- [ ] your Tests are failing please make sure that they are passing in order to complete the requirements 
<img width="913" alt="Screenshot 2023-02-18 at 21 56 24" src="https://user-images.githubusercontent.com/61790587/219899137-5fac3aaa-1f7d-4c8d-873f-10ef2253fd21.png">